### ISSUE TITLE
Classes cause program to be identified as "Wacapew.C!ml" by windows defender

### ISSUE THREAD

**Issue by Sokoloft** [TomSchimansky/CustomTkinter#2260]
Hello,

Since switching my gui file to classes. As per instruction of the wiki. Its detected as "Wacapew.C!ml" by windows defender. I specifically just reverted one of my old versions from classes to no classes aside from (ctk.Toplevel). Is it an issue specifically with making a Root class? Like `class Root(ctk.CTk):`

**Comment by Sokoloft** [comment]
Okay. It isn't classes like I thought it was. The issue is I was importing sys and os separately. 

```import pyperclip, os, sys, write```

My old versions of code used the following, which imports sys and os from customtkinter.

```from customtkinter import *```

I'll determine which library is actually causing the issue out of the two. Then make a follow up post on the issue libraries github.

**Comment by orztrickster** [comment]
It is common for files compiled from python using pyinstaller to be incorrectly identified as malware in this way.
You can repay.
https://www.microsoft.com/en-us/wdsi/filesubmission

**Comment by Anonymous6598** [comment]
Yeah, I also had this problem. One day, I remember, all my projects, which had been recompiled to exe, were deleted by windows defender. You should fix that.

**Comment by Sokoloft** [comment]
I don't know what the cause of the problem is. It mainly seems to be when things are formatted inside of classes. Without classes, my code would not be nuked by windows defender as a false positive. 

I could change the code around some and it would not get picked up by defender initially. However adding the next build to the system without changing the imports or code I changed to get it to not show up. It would show up as a false positive. 

I guess the only solution is as @orztrickster mentioned. To submit it to MS as a false positive so they can train their AI to not detect it. I plan to do that once I get the code here on github.

**Comment by Anonymous6598** [comment]
> I don't know what the cause of the problem is. It mainly seems to be when things are formatted inside of classes. Without classes, my code would not be nuked by windows defender as a false positive.
> 
> I could change the code around some and it would not get picked up by defender initially. However adding the next build to the system without changing the imports or code I changed to get it to not show up. It would show up as a false positive.
> 
> I guess the only solution is as @orztrickster mentioned. To submit it to MS as a false positive so they can train their AI to not detect it. I plan to do that once I get the code here on github.

I hope so
